util/cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mc-util-cli"
-version = "0.1.0"
+version = "1.3.0-pre0"
```

Comment:
Alternatively, we could lower the version of `mc-util-build-info`, since it does not have any other `mc-*` deps.

Reply:
Tracking the release version seems a viable strategy